series_renamer/series_renamer.py

```diff
@@ -247,37 +254,41 @@ def main(path='.'):
 
 	if x == 'y':
 		for i in renames.items():
-			if os.path.isfile(path + '/' + i[1]):
+			if os.path.isfile(i[1]):
 				warn('File {0} exists, skipping'.format(i[1]))
 			else:
-				os.rename(path + '/' + i[0], path + '/' + i[1])
-				subtitleRename(path + '/' + i[0], path + '/' + i[1])
+				os.rename(i[0], i[1])
+				subtitleRename(i[0], i[1])
 		print('Renaming Successful')
 
 	os.remove(logfile)
 	return 0
 
 
-def getNums(path):
+def getNums(path, recurse=False):
 	"""
 	Scans the path, looks for series files and gets contenders of the episode numbers and season numbers.
 	Stores them in epns
 	"""
 
 	exts = ['mkv', 'mp4', 'avi', 'flv', 'mpg', 'mpeg', 'wmv', 'webm', 'vob', 'mov', '3gp', 'ogv']
 
-	for i in os.listdir(path):
-		if not os.path.isfile(path + '/' + i):
+	for root, dirs, files in os.walk(path):
```

Comment:
I like suggestion 1 but that would require learning more about os.walk . BTW, I don't think users will store other things in a TV Series collection folder. If the folder is for "The West Wing", it will generally contain items related to "The West Wing" only and so at max 1k-2k items. 
So I think this optimization, though theoritically justified, is not [practically] needed.